File: apps/ios/Barns/BarnsApp/Domain/Entities/WallGreenPattern.swift
import Foundation

struct WallGreenPattern: Equatable, Identifiable, Sendable {
    let id: String
    let name: String
    let recommendedSpace: String
    let mood: String
    let maintenanceLevel: MaintenanceLevel
    let description: String
    let imageUrl: URL?
}

enum MaintenanceLevel: String, Equatable, Sendable {
    case low
    case standard
    case medium
    case high
}

File: apps/android/app/src/main/java/com/barns/app/domain/model/WallGreenPattern.kt
package com.barns.app.domain.model

data class WallGreenPattern(
    val id: String,
    val name: String,
    val recommendedSpace: String,
    val mood: String,
    val maintenanceLevel: MaintenanceLevel,
    val description: String,
    val imageUrl: String?,
)

enum class MaintenanceLevel { LOW, STANDARD, MEDIUM, HIGH }
